utils/result_try.hh

```diff
@@ -129,6 +141,18 @@ public:
     void forward_to_promise(seastar::promise<S>& p) {
         p.set_exception(std::move(_eptr));
     }
+
+    const std::exception_ptr& as_inner() const & {
+        return _eptr;
+    }
+
+    std::exception_ptr&& as_inner() && {
+        return std::move(_eptr);
+    }
+
+    std::exception_ptr clone_inner() {
+        return _eptr;
+    }
```

Comment:
Would `clone_inner() &&` make sense?

Reply:
Wouldn't that imply that the exception is moved out of the handle? The purpose of `clone_inner` is to get an instance of exception_container/exception_ptr without consuming the one in the handle.

Reply:
Yes of course, if it's an rvalue you'd just use it.

Reply:
Hmm, but then it wouldn't technically do any cloning, so I think the name would be confusing. Could you suggest a better name?

Reply:
I meant I agree that `clone_inner() &&` is useless.